### PR TITLE
fix: correct steps for pages router migration

### PR DESCRIPTION
**Pull request by grixu**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSDoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by mateuszo — COMMENTED** [review]
Some small changes required. With the suggested fix, it worked locally. However, the deployed version fails. 
live URL: https://extensibility-demo.europe-west1.gcp.storefrontcloud.io/
instance: https://console.alokai.com/organizations/490/instances/extensibility-demo-europe-west1-gcp-storefrontcloud-io/overview/insights
logs:
[logs (1).json](https://github.com/user-attachments/files/16313416/logs.1.json)

File: docs/content/4.sdk/6.migration-guide.md [hunk 1]
```diff
@@ -27,122 +68,191 @@ NEXT_DEFAULT_HTML_CACHE_CONTROL="public, max-age=0, s-maxage=15, must-revalidate
...
+export const { getSdk } = createSdk(options, ({ buildModule, middlewareModule, config, getRequestHeaders }) => ({ // [!code ++]
+  unified: buildModule(middlewareModule<UnifiedEndpoints>, {
+    apiUrl: `${middlewareUrl}/commerce`, // [!code --]
+    apiUrl: `${config.middlewareUrl}/commerce`, // [!code ++]
+    cdnCacheBustingId: config.cdnCacheBustingId, // [!code ++]
+
+    defaultRequestConfig: {
+      headers: getRequestHeaders(),
+    },
+    methodsRequestConfig: config.defaultMethodsRequestConfig.unifiedCommerce.middlewareModule, // [!code ++]
```
[the remark this answers is not in the document]
Comment (grixu) [reply docs/content/4.sdk/6.migration-guide.md]: Removed — this change requires also change on the middleware. So I would not mix the cases in this Migration Guide.

File: docs/content/4.sdk/6.migration-guide.md [hunk 2]
```diff
@@ -27,122 +68,191 @@ NEXT_DEFAULT_HTML_CACHE_CONTROL="public, max-age=0, s-maxage=15, must-revalidate
...
+
+  const params = ['f_auto', 'c_limit', 'dpr_auto', `w_${width}`, `q_${quality || 'auto'}`];
+
+  return `${baseURL}${params.join(',')}/${src}`;
+};
+
+export default cloudinaryLoader;
+```
+
+7. Update `.gitignore`. Add this line at the bottom of the file:
```
[the remark this answers is not in the document]
Comment (grixu) [reply docs/content/4.sdk/6.migration-guide.md]: Done

File: docs/content/4.sdk/6.migration-guide.md [hunk 3]
```diff
@@ -323,7 +396,7 @@ echo "envs GIT_SHA: $GIT_SHA" # [!code ++]
 node ./server/index.mjs
 ```
 
-6. Update the deployment script
+11. Update the deployment script
```
[the remark this answers is not in the document]
Comment (grixu) [reply docs/content/4.sdk/6.migration-guide.md]: Done

File: docs/content/4.sdk/6.migration-guide.md [hunk 4]
```diff
@@ -2,9 +2,50 @@
...
+// ... rest of the config
+const withPwa = require('next-pwa')({
+// ...
+});
+
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
+  // ...
+  images: env('NEXT_PUBLIC_IMAGE_LOADER_FETCH_URL') ? cloudinaryConfig : defaultImageConfig, // [!code ++]
```
[the remark this answers is not in the document]
Comment (grixu) [reply docs/content/4.sdk/6.migration-guide.md:30]: Great, thanks. Fixed

**Review by mateuszo — APPROVED** [review]

**Pull request merged**